### PR TITLE
Issue #365 documentation about rebuilding performance counter libraries

### PR DESCRIPTION
**Pull request by ramonsmits**

### CODE REVIEW

File: nservicebus/nservicebus-fails-or-hangs-during-performance-counter-setup.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Resolution
+
+The performance counters libraries need to be rebuild by doing the following steps:
+
+1. Open an elevated command prompt
+2. Execute the following command rebuilds the performance counter libraries:
+
+    `LODCTR /R`  (Mind the uppercase /R)
+
```
Comment (SimonCropp) [inline nservicebus/nservicebus-fails-or-hangs-during-performance-counter-setup.md]: 3 and 4 are not realky steps. Just two extra points if the above two steps dont work.

**Comment by ramonsmits** [comment]
@simoncropp Is this structure better?

**Comment by SimonCropp** [comment]
@ramonsmits yep

File: nservicebus/nservicebus-fails-or-hangs-during-performance-counter-setup.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+> Due to the corruption you may have to re-install performance counters of NServiceBus or other products. The following articles help in restoring the NServiceBus performance counters:
+>
```
Comment (SimonCropp) [inline nservicebus/nservicebus-fails-or-hangs-during-performance-counter-setup.md]: why the `>` characters? are these lines a quote?

Comment (ramonsmits) [inline nservicebus/nservicebus-fails-or-hangs-during-performance-counter-setup.md]: @SimonCropp Yes, to make it stand out. Sort of a note. Is there a way to make a note multi line?

Comment (SimonCropp) [inline nservicebus/nservicebus-fails-or-hangs-during-performance-counter-setup.md]: https://github.com/Particular/docs.particular.net#multi-line

**Comment by SimonCropp** [comment]
Note that since this page does not exist in the menu and is not linked to from any page it will only be discoverable via search.

**Comment by ramonsmits** [comment]
Thanks @SimonCropp as it indeeds needs to be added to the index. I now added it as an item to the FAQ section but somehow could be better. Maybe adding a 'Troubleshooting' or 'Issues' section?

**Pull request merged**